### ISSUE TITLE
Update Quick sub fields for Methods and Orgs - Pre Montreal

### ISSUE THREAD

**Issue by jesicarson** [participedia/frontend#408]
**Quick Submit for Methods**
Name of method [limited word count]
Purpose: Which civic, governance, or political purposes is this method designed to address? – [Use existing list of purposes from .net, select one or multiple]
Explanatory text: “Please feel free to indicate more than one possible purpose where appropriate.”
 
**Quick Submit for Organizations**
Name of organization [limited word count]
Location

**Comment by jesicarson** [comment]
Also include tags in quick submit, so we can tag entries as 'oidp2017' to show up in search results.

**Comment by davidascher** [comment]
@jesicarson which field in the .net site is the 'purpose' mapping to?  I'm not seeing that word used in the method form.  I see the bit about purpose of _communication_, but that feels different.

**Comment by jesicarson** [comment]
I've emailed Pat to ask what he'd like to do, as he provided these sections. I assume for now just leave it out. But in review I also realized I missed some things:

- [ ] Methods and Orgs should also include media 

- [ ] Orgs should include a field where someone can enter a url (ie: "Organization website")

**Comment by davidascher** [comment]
- [x] methods and orgs do have photos
- [ ] we need to add video URLs to all.
- [ ] URLs for orgs should be in the schema, in which case it's trivial to add to the form

**Comment by jesicarson** [comment]
I think all quick submits should include a place for an outside url(s). How about: "Relevant Links" ?
@davidascher Is this a problem if there's no place for URLs in the existing schema? Because I'm pretty sure there isn't, currently there's only a section for "Links" as a header in the body text narrative for all types.

**Comment by davidascher** [comment]
I agree, if there's no URLs in the current schema we just need to add one, it's high value and easy.

**Comment by davidascher** [comment]
@dethe do we have a schema slot for URLs (in each of case, method, org)?

**Comment by dethe** [comment]
No, but I'll add `url` to the migration I'm working on.

**Comment by dethe** [comment]
All the types have an array for videos and one for files.

**Comment by jesicarson** [comment]
Closes: https://github.com/participedia/frontend/issues/306

**Comment by dethe** [comment]
@jesicarson Neither purpose nor explanatory text are in our current model. Is it important to add those to our backend model before user testing. The rest of this issue is done.

**Comment by jesicarson** [comment]
@dethe yes, any further updates to forms can wait until after user testing, in fact they can wait until we have Pat/Grahams final proposal and we've decided on what gets changed. Let's do that all at once, later.

**Comment by dethe** [comment]
Alright, I'll leave this open until we have a new issue to replace it, but I'll pull the "before user testing" flag off.
